vcd/datasource_vcd_vcenter.go

```diff
@@ -2,96 +2,91 @@ package vcd
 
 import (
 	"context"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
-	"log"
-	"net/url"
+	"fmt"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/vmware/go-vcloud-director/v3/govcd"
 	"github.com/vmware/go-vcloud-director/v3/types/v56"
 )
 
+// TODO: TM: validate compatibility with old data source
+
 func datasourceVcdVcenter() *schema.Resource {
 	return &schema.Resource{
-		ReadContext: datasourceVcenterRead,
+		ReadContext: datasourceVcdVcenterRead,
+
 		Schema: map[string]*schema.Schema{
 			"name": {
 				Type:        schema.TypeString,
 				Required:    true,
-				Description: "Name of vCenter.",
+				Description: fmt.Sprintf("Name of %s", labelVirtualCenter),
 			},
-			"vcenter_version": {
+			"url": {
 				Type:        schema.TypeString,
 				Computed:    true,
-				Description: "vCenter version",
+				Description: fmt.Sprintf("URL including port of %s", labelVirtualCenter),
```

Comment:
nit, but sounds better
```suggestion
				Description: fmt.Sprintf("URL of %s, including port", labelVirtualCenter),
```

Reply:
Very good you noticed this. It doesn't have a port anymore as I made a workaround there https://github.com/vmware/terraform-provider-vcd/pull/1343/files#diff-107949ff5e9f5d11e42e09e2c609bbf3e4c722f7f3e89ae8b11a09e9521aba56R182

Changed wording as well